integrations/google-calendar/actions/settings.ts

```
@@ -0,0 +1,23 @@
+import type { NangoAction, SettingsResponse, CalendarSetting } from '../../models';
+import type { GoogleCalendarSettingsResponse } from '../types';
+
+export default async function runAction(nango: NangoAction): Promise<SettingsResponse> {
+    const settings: CalendarSetting[] = [];
+    let pageToken: string | undefined;
+
+    do {
+        const { data: response } = await nango.get<GoogleCalendarSettingsResponse>({
+            endpoint: '/calendar/v3/users/me/settings',
+            params: pageToken ? { pageToken: pageToken } : ''
+        });
```

ESLint

[error] Nango API calls should include a retries property in the options object

(@nangohq/custom-integrations-linting/proxy-call-retries)